### PR TITLE
update requirements

### PR DESCRIPTION
**Pull request by mparrSaatva**
- Remove items from “Bonus” that we actually DO want to see (can vary  by level)
- Require a “from scratch” component to ensure they demonstrate their css/html abilities (mid and senior)
- Provide soft guidance around how we’re evaluating (break it up into Functionality/UI/Creativity)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## Mattress Shop Page
+Create a single page React application using the screen shot provided (mock.png) as your guide. Please use the data provided in mattresses.json to populate the mattress data in the application.
```
Comment (mparrSaatva) [inline README.md:2]: how do we message "Create React App" ?

"We recommend using Create React App, but if you roll your own build, please include the node version or an .nvmrc file and any other instructions or dependencies."

File: README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+### Get Creative!
+The following requirements are loose - this is an opportunity to make the app your own!
+1. Display the Review Rating for the selected mattress.
+2. Add an animation where you see fit.
+
+### Bonus Items
+1. Use a css preprocessor if you prefer sass or less.
+2. Add your own flair to the app.
+3. Add a unit test for the "Add to Cart" functionality.
+
```
Comment (mparrSaatva) [inline README.md:27]: Suggested breakdown of items by level:

Eng I:
all bonus (review and animation)

Eng II:
review and animation are required

Eng III:
review, animation, unit test for add to cart (create react app comes with a unit test already so it shouldn't be THAT hard to set up a single interaction test, even if someone isn't familiar with testing?)

File: README.md (hunk 1 above)
Comment (jpostensaatva) [reply README.md:2]: Sounds good to me.

Comment (AnnieH) [reply README.md:2]: I like it. We may want to add a link to Create React App on the off chance that someone is not familiar with it.

**Review by AnnieH — APPROVED** [review]

**Review by jpostensaatva — APPROVED** [review]

**Pull request merged**